spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Level.java

```diff
@@ -27,9 +26,8 @@
      */
     NONE;
 
-    @Override
-    public String toJSONString() {
-        return String.format("\"%s\"", name().toLowerCase());
+    public String toJsonString() {
+        return name().toLowerCase();
```

Comment:
Do we need this method? The old one was provided to configure the org.json's serializer.
So I assume what we need is not this method but similar way to configure the serialier, like [SerializedName](https://stackoverflow.com/a/24621292).